examples/spring-streams-example/src/main/kotlin/stove/spring/streams/example/kafka/CustomSerDer.kt

```diff
@@ -0,0 +1,28 @@
+package stove.spring.streams.example.kafka
+
+import com.google.protobuf.Message
+import io.confluent.kafka.schemaregistry.testutil.MockSchemaRegistry
+import io.confluent.kafka.serializers.AbstractKafkaSchemaSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG
+import io.confluent.kafka.streams.serdes.protobuf.KafkaProtobufSerde
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.core.env.Environment
+import org.springframework.stereotype.Component
+
+@Component
+class CustomSerDer(
```

Comment:
`CustomSerDe` I believe, there might be a typo.

Reply:
fixed